### PR TITLE
Destroy outline of attack targets after cooldown

### PR DESCRIPTION
**Pull request by hhliow**
After the mesh cooldown on an attack target, destroy (rather than just hiding) the outline.
This should not affect block mesh cool downs.

When an outline is hidden, the mesh it is on can still occlude other objects outlines even if those objects are closer the player.

### CODE REVIEW

File: ValheimVRMod/Scripts/AttackTargetMeshCooldown.cs [hunk 1]
```diff
@@ -7,6 +7,11 @@ public class AttackTargetMeshCooldown : MeshCooldown {
         public static bool staminaDrained;
         private static MeshCooldown lastAttackTargetMeshCooldown;
 
+        void Awake()
+        {
+            keepOutlineInstance = false;
```
Comment (brandonmousseau) [inline ValheimVRMod/Scripts/AttackTargetMeshCooldown.cs:12]: Seems like this variable is no longer used? In MeshCooldown, it only the `keepOutlineInstance()` function is being used, which is hardcoded to return `true`.

Comment (hhliow) [reply ValheimVRMod/Scripts/AttackTargetMeshCooldown.cs]: Meant to remove this. Thanks!

Comment (brandonmousseau) [reply ValheimVRMod/Scripts/AttackTargetMeshCooldown.cs]: I guess I probably just am not familiar with the syntax. Is
```cs
keepOutlineInstance = false;
```

the same thing as just overriding the original function definition with this?
```cs
bool keepOutlineInstance()
{
    return false;
}
```

Comment (brandonmousseau) [reply ValheimVRMod/Scripts/AttackTargetMeshCooldown.cs]: Oh ok, thanks!

**Review by brandonmousseau — APPROVED** [review]

**Pull request merged**